chacra/tests/controllers/repos/test_repos.py

```diff
@@ -13,7 +13,7 @@ def test_repo_exists(self, session):
         )
         repo.path = "some_path"
         session.commit()
-        result = session.app.get('/repos/foobar/firefly/ubuntu/trusty/api/')
+        result = session.app.get('/repos/foobar/firefly/ubuntu/trusty/')
```

Comment:
Why did you need to change this? Did this change break the /api endpoint or do you plan for that endpoint to no longer be needed because of nginx serving everything at /r/?